### PR TITLE
02 creation pokemon infos

### PR DESCRIPTION
**Pull request by AntoineCAU**

### CODE REVIEW

**Review by gasparddhi — DISMISSED** [review]
Changing the structure to a router okay

**Comment by LoicBrassart** [comment]
Require a validation from your SM before you assign me plz ^^

**Review by gasparddhi — DISMISSED** [review]
Everything is ok for me

File: src/components/PokemonPage.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import './css/PokemonPage.scss';
+
+const PokemonPage = (props) => {
+  const [infos, setinfos] = useState({
+    abilities: [
+      {
+        ability: {
+          name: 'Fatalfoutre',
```
Comment (LoicBrassart) [inline src/components/PokemonPage.jsx]: NOEZ ! （╯°□°）╯︵( .o.)

File: src/components/PokemonPage.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+          name: 'undefined',
+        },
+      },
+    ],
+  });
+
+  useEffect(() => {
+    axios
+      .get(
+        `https://pokeapi.co/api/v2/pokemon/${props.match.params.pokemonName}`
```
Comment (LoicBrassart) [inline src/components/PokemonPage.jsx:37]: You should use hooks (`useParams()` from `react-router-dom`)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
No, no, nonono ! >_<

**Review by gasparddhi — APPROVED** [review]
Code très propre j'approuve très beau travail bravo !

**Review by gasparddhi — APPROVED** [review]
well done

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**